forge/ee/lib/billing/trialTask.js

```diff
@@ -108,7 +108,11 @@ module.exports.init = function (app) {
             }
         })
         for (const project of trialProjects) {
-            await app.billing.addProject(team, project.Project)
+            if (project.Project.state !== 'suspended') {
```

Comment:
[nb] Notę to self, another place where a `app.billing.projectBilling(project).shouldBeBilled()` wrapper would be of value.